resources/tags.yaml

```diff
@@ -0,0 +1,171 @@
+  description: 'Charting'
+cloud_platform: 
+  title: 'Cloud Platform'
+  description: 'Offers hosting on the same supporting infrastructure'
+cms: 
+  title: 'CMS'
+  description: 'Content Management System'
+collaboration: 
+  title: 'Collaboration'
+  description: 'Collaboration'
+crm: 
+  title: 'CRM'
+  description: 'Customer Relationship Management'  
+customer_support:
+  title: 'Customer Support'
+  description: 'Range of customer services to assist users of a product'
+dashboard: 
+  title: 'Dashboard'
+  description: 'Control Panel'
+database:
+  title: 'Database'
+  description: 'Collection of schemas, tables, queries, reports, views and other objects'
+developer_tools: 
+  title: 'Developer Tools'
+  description: 'Developer Tools'
+digital_signature:
+  title: 'Digital Signature'
+  description: 'Scheme for demonstrating the authenticity of a digital message or documents'
+ecommerce:
+  title: 'eCommerce'
+  description: 'Facilitation of trading in products or services using computer networks'  
+education:
+  title: 'Education'
+  description: 'Education'
+email: 
+  title: 'Email'
+  description: 'Email'
+enterprise: 
+  title: 'Enterprise'
+  description: 'Enterprise'  
+erp:
+  title: 'ERP'
+  description: 'Enterprise resources planning'
+event_management:
+  title: 'Event Management'
+  description: 'Event Management'
+file_management: 
+  title: 'File Management'
+  description: 'File Management'  
+financial:
+  title: 'Financial'
+  description: 'Financial'
+forms: 
+  title: 'Forms'
+  description: 'Forms'
+hosting: 
+  title: 'Hosting'
+  description: 'Hosting'
+image:
+  title: 'Image'
+  description: 'Image'
+iot:
+  title: 'IoT'
+  description: 'Internet of Things'
+issue_tracking: 
+  title: 'Issue Tracking'
+  description: 'Issue Tracking'
+jobs:
+  title: 'Jobs'
+  description: 'Jobs'
+lead_generation: 
+  title: 'Lead Generation'
+  description: 'Lead Generation'
+location:
+  title: 'Location'
+  description: 'Location'
+machine_learning:
+  title: 'Machine Learning'
+  description: 'Pattern recognition and computational learning theory'
+marketing: 
+  title: 'Marketing'
+  description: 'Marketing'  
+messaging:
+  title: 'Messaging'
+  description: 'Messaging'
+mobile: 
+  title: 'Mobile'
+  description: 'Mobile'    
+monitoring:
+  title: 'Monitoring'
+  description: 'Monitoring'
+nlp: 
+  title: 'NLP'
+  description: 'Natural Language Processing'  
+notifications:
+  title: 'Notifications'
+  description: 'Notifications'
+office: 
+  title: 'Office'
+  description: 'Office'
+package_management:
+  title: 'Package Management'
+  description: 'Package Management'
+payment: 
+  title: 'Payment'
+  description: 'Payment Processing'  
+personal_information_management:
+  title: 'Personal Information Management'
+  description: 'Personal Information Management'
+project_management: 
+  title: 'Project Management'
+  description: 'Project Management'  
+real_estate:
+  title: 'Real Estate'
+  description: 'Real Estate'
```

Comment:
I don't think APIs.guru have something like that.
We could think about it when such APIs will appear.